src/web/mjs/connectors/Tapas.mjs

```diff
@@ -14,11 +14,16 @@ export default class Tapas extends Connector {
     }
 
     async _getMangaFromURI(uri) {
-        let request = new Request(uri, this.requestOptions);
-        let data = await this.fetchDOM(request, 'div.info a.title');
-        let id = uri.pathname;
-        let title = data[0].textContent.trim();
-        return new Manga(this, id, title);
+        let spliced_url = (await this.fetchDOM(new Request(uri), 'meta[property="al:android:url"]'))[0].content.split('/')
+        // title can be in two formats: tapas/series/<series_name> or tapas/series/<series_name>/info
+        let seriesId = (spliced_url[spliced_url.length - 1] == 'info' ? spliced_url[spliced_url.length-2] : spliced_url.pop())
```

Comment:
are you sure the meta url can get the /info at the end?

Reply:
Yes, I had the issue while testing.

Reply:
ok, ill have to fix that on my end too :D

you could have just replaced ending /info by nothing, tbh. It would have been more readable.

`content.replace(/\/info$/, '').split('/')`

Reply:
Did the first one, what kind of linter should I be using for ts?

Reply:
eslint. Integration should have displayed the errors but fixes are needed on CI. Gotta fix it myself if you cant anyway